Source/rclUE/Public/Srvs/ROS2SpawnEntitiesSrv.h

```diff
@@ -0,0 +1,254 @@
+// Copyright 2021 Rapyuta Robotics Co., Ltd.
+// This code has been autogenerated from UE_msgs/SpawnEntities.srv - do not modify
+
+#pragma once
+
+#include <CoreMinimal.h>
+
+#include "Srvs/ROS2GenericSrv.h"
+#include "rclcUtilities.h"
+#include "ue_msgs/srv/spawn_entities.h"
+
+#include "ROS2SpawnEntitiesSrv.generated.h"
+
+// potential problem: if this struct is defined multiple times!
+USTRUCT(Blueprintable)
+struct RCLUE_API FROSSpawnEntities_Request
+{
+	GENERATED_BODY()
+
+public:
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> type;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> name;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FVector> position;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FQuat> orientation;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FVector> twist_linear;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FVector> twist_angular;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> reference_frame;
+
+	UPROPERTY(EditAnywhere, BlueprintReadWrite)
+	TArray<FString> tags;
```

Comment:
```suggestion
	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FString> TypeList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FString> NameList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FVector> PositionList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FQuat> OrientationList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FVector> TwistLinearList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FVector> TwistAngularList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FString> ReferenceFrameList;

	UPROPERTY(EditAnywhere, BlueprintReadWrite)
	TArray<FString> Tags;
```
This is UE adapter class, not the directly generated cpp from .srv file, so we should use UE convention for clarity I suppose.